Modules/_testcapi/dict.c

```diff
@@ -165,15 +252,21 @@ dict_getitemref(PyObject *self, PyObject *args)
     }
 }
 
+
+/*[clinic input]
+_testcapi.dict_getitemstringref
+    obj: object
+    value: object
```

Comment:
Ditto. `value` is not part of the spec of this method; `dict_getitemstringref` only takes two positional-only arguments.